static-lang-word-lists/data/diffenator/Common.txt

```diff
@@ -0,0 +1,11 @@
+0,,,zero
```

Comment:
Is this a word list, or some kind of CSV data?

Reply:
Maybe it's an issue for upstream, not to be fixed here if our goal here to ship diffenator lists as they are